lib/mailjex/utils/comms.ex

```diff
@@ -14,9 +15,16 @@ defmodule Mailjex.Utils.Comms do
     hdrs = [body: Poison.encode!(body)]
     |> headers
 
-    path
-    |> api_url
-    |> HTTPotion.post(hdrs)
+    if @development_mode do
+      IO.puts "Development Mode Enabled"
```

Comment:
Could be worth making this "Mailjex Development Mode Enabled" - so people know where its coming from in their logs. Also, same for the Body and Header labels below - prefix with "Mailjex".